cfme/utils/mgmt_system/__init__.py

```diff
@@ -10,7 +10,11 @@
 from cfme.utils.ssh import SSHClient
 
 # Overrides
-from ovirtsdk.xml import params
+try:
+    from ovirtsdk.xml import params
+except ImportError:
+    # there is a hack in place to survive this missing in wrapanapi itself
```

Comment:
Can you explain a little further in your comment here on why it is ok if we 'pass' on this import? If we don't import `params` it seems to me lot of code would fail in the below RHEVMSystem class...